Resources/Prototypes/Entities/Mobs/Player/ratvar.yml

```diff
@@ -0,0 +1,80 @@
+- type: entity
+  abstract: true
+  id: MobRatvarBase #can't have the clockwork justiciar be associated with his (im)mortal enemy
```

Comment:
make a spawning animation too tg should have it

Reply:
tg doesn't have a spawning animation, I'll make one.

Reply:
Citadel might have one

Reply:
I checked the citdael github and they don't have a spawning animation for ratvar